src/molecules/Component.h

```diff
@@ -93,6 +93,9 @@ class Component {
 	void addQuadrupole(double x, double y, double z,
 	                   double eQx, double eQy, double eQz, double eQabs);
 
+	/** functions to fix the LJTS after rc is read in the simulation*/
+	void updateAllLJcenters(double rc);
```

Comment:
This function only updates the shift. So call it something like `updateAllLJcentersShift()`. Otherwise, it's not immediately clear what is updated.

Reply:
Yep, fixed